components/molecules/Breadcrumb.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+import BreadcrumbMenu from './BreadcrumbMenu';
+
+const baseClass = 'gds-page-header__breadcrumbs-list-item';
+const filterMenu = pathname => ({ path }) => path.charAt(0) !== ':' && !pathname.includes(path);
+
+const Breadcrumb = props => {
+    const {
+        config: { title, path, subpaths },
+        LinkElt,
```

Comment:
also feels weird to have a capitalized prop. I'd prefer it renamed when destructured.

Reply:
Fixed